mailserver/mailserver_test.go

```diff
@@ -58,6 +61,70 @@ func TestDBKey(t *testing.T) {
 	assert(byte(i/0x1000000) == k.raw[0], "big endian expected", t)
 }
 
+func TestMailServer(t *testing.T) {
+	var server WMailServer
+
+	setupServer(t, &server)
+	defer server.Close()
+
+	rand.Seed(seed)
```

Comment:
Experienced a time based seed in tests as a bad solution due to non-repeatability. Normally in tests no real random is needed.